.gitignore

```diff
@@ -6,27 +6,49 @@ frontend/package-lock.json
 frontend/.pnp
 frontend/.pnp.js
 
+backend/node_modules
```

Comment:
not that it makes too much difference, but wouldn't */node_modules work

Reply:
yeah i like that better